drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -353,14 +352,18 @@ def do_attempt(self, data, suffix=''):
         self._mark_complete_and_publish_grade()  # must happen before _get_feedback
 
         overall_feedback_msgs, misplaced_ids = self._get_feedback()
+        misplaced_items = [self._get_item_definition(int(item_id)) for item_id in misplaced_ids]
```

Comment:
@staubina yes, as far as I know, `self.item_state` stores items' IDs as strings, while `self.data['items']` uses integers. I don't think this is a good time to modify it so both `self.data['items']` and `self.item_state` use ints, as those will affect existing DnDv2 instances.